README.md

```diff
@@ -158,6 +160,36 @@ This matches the specification for ES module workers, supporting all features of
 
 > Module workers are only supported in browsers that provide dynamic import in worker environments, which is only Chrome currently.
 
+### Fetch Hook
+
+The ES Module Shims fetch hook can be used to implement transform plugins.
+
+For example:
+
+```js
+importShim.fetch = async function (url) {
+  const response = await fetch(url);
+  if (response.url.endsWith('.ts')) {
+    const source = await response.body();
+    const transformed = tsCompile(source);
+    return new Response(URL.createObjectURL(new Blob([transformed], { type: 'application/javascript' })), { status: 200 });
```

Comment:
`URL.createObjectURL` is not needed (it returns a URL which would be used as contents of the response)
And the `status` is `200` by default, so it doesn't have to be explicitly specified

```js
return new Response(new Blob([transformed], { type: 'application/javascript' });
```